src/program.js

```diff
@@ -503,6 +503,11 @@ Example: $0 --help run.
         demandOption: false,
         type: 'boolean',
       },
+      'binary-args': {
```

Comment:
Using `binary-args` here implies that the command line becomes `--binary-args`. That is a bit too long to my taste. Could you rename this to `args`
and add an `alias: ['arg'],`

I would also have liked the `'a'` shorthand, but unfortunately the letter is already taken by the `--artifacts-dir` command.